format/expr.go

```diff
@@ -22,6 +22,45 @@ type printer struct {
 
 func (p *printer) expr(e expr.Expr) {
 	switch e := e.(type) {
+	case *expr.Binary:
+		WriteExpr(p.buf, e.Left)
+		p.buf.WriteString(e.Op.String())
+		WriteExpr(p.buf, e.Right)
+	case *expr.Unary:
+		p.buf.WriteString(e.Op.String())
+		WriteExpr(p.buf, e.Expr)
+	case *expr.Bad:
```

Comment:
Before these would fall through to the default case and print the data structure. I think that's still useful. How about commenting this block of cases out and putting a TODO on it?

Reply:
done.